example/lib/main.dart

```diff
@@ -302,7 +302,7 @@ class _MyHomePageState extends State<MyHomePage> {
       }
 
     } on PlatformException catch (e) {
-      print('${e.message}, ${e.details}');
+      print('Error Message: ${e.message}, Error Details: ${e.details}, Error Code: ${e.code}');
       if (mProgressBarUtil.isProgressBarShowing()) {
         mProgressBarUtil.hideProgressDialog();
```

Comment:
Would we be able to show how we would want the errors to be handled by using the `httpStatusCode`?